### DISCUSSION TITLE
DateTimeInput

### DISCUSSION THREAD

**Discussion by lorypelli** [mantinedev/mantine#2105] [Feature requests]
Please add an option to add a time input in the calendar like on this codesandbox https://codesandbox.io/embed/g8b7k

**Comment by rtivital** [comment]
DateTimePicker is a complex component, I'm not planning to it soon, it maybe a part of 6.0 release

**Reply by lorypelli** [reply]
Ok, thanks

**Reply by lorypelli** [reply]
> DateTimePicker is a complex component, I'm not planning to it soon, it maybe a part of 6.0 release

@rtivital I think v6 will soon be released, now it's v5.9.3, so, what about this?

**Reply by rtivital** [reply]
You can use the preview package https://rtivital.github.io/mantine-dates-6/
I haven't started the work on 6.0 yet. If version now is 5.9, it does not mean that next version will be 6.0, it can be 5.10, then 5.11 and so on.
